docs/02-sygma-sdk/03-Quick-Start/02-transfer-token.md

```diff
@@ -7,4 +7,56 @@ sidebar_position: 2
 draft: true
 ---
 
-More to come soon...
\ No newline at end of file
+Transferring a token with `Sygma SDK` is very straigthforward. You need to build the transfer object to pass it to your signer to send the transaction.
```

Comment:
We need to somehow highlight this is relevant only for sending from EVM based chains, probably having title something like this: **Transferring tokens from EVM based chains**

Then we would have new title for similar substrate snippet on this page.

Reply:
Two different sections (one for EVM and one for Substrate) separated by a linebreak + heading highly recommended for when we have the Substrate stuff down as well

Reply:
Done